extensions/ql-vscode/src/databases/config/db-config-validator.ts

```diff
@@ -9,22 +9,22 @@ import {
 } from "../db-validation-errors";
 
 export class DbConfigValidator {
-  private readonly schema: any;
+  private readonly validateSchemaFn: ValidateFunction;
 
   constructor(extensionPath: string) {
     const schemaPath = resolve(extensionPath, "databases-schema.json");
-    this.schema = readJsonSync(schemaPath);
+    const schema = readJsonSync(schemaPath);
```

Comment:
Can we also "cache" the JSON file by importing it instead of reading it from disk? I believe you should be able to do something like `import * as schema from "../../../databases-schema.json";`

Reply:
Unfortunately not atm because the schema file is outside of `/src` so TS can't see it. We can potentially move it, but I'll leave that for another day.